js/bundle.go

```diff
@@ -252,22 +252,21 @@ func (b *Bundle) getExports(logger logrus.FieldLogger, rt *goja.Runtime, options
 }
 
 // Instantiate creates a new runtime from this bundle.
-func (b *Bundle) Instantiate(logger logrus.FieldLogger, vuID uint64) (bi *BundleInstance, instErr error) {
-	// TODO: actually use a real context here, so that the instantiation can be killed
-	// Placeholder for a real context.
-	ctxPtr := new(context.Context)
-
+func (b *Bundle) Instantiate(
+	logger logrus.FieldLogger, vuID uint64, vuImpl *moduleVUImpl,
```

Comment:
```suggestion
	logger logrus.FieldLogger, vuID uint64, vuImpl *moduleVUImpl,
```
Could now we have `vuID` as part of `vuImpl`?

Reply:
probably in another PR see https://github.com/grafana/k6/pull/2373#discussion_r799485932